go/mysql/conn.go

```diff
@@ -613,6 +613,11 @@ func (c *Conn) ID() int64 {
 	return int64(c.ConnectionID)
 }
 
+// Ident returns a useful identification string for error logging
+func (c *Conn) Ident() string {
```

Comment:
Could we just use String() here, so we can just print the connection as:
'%v', c
in the code?